components/social/org.wso2.carbon.social.core/src/main/java/org/wso2/carbon/social/core/service/SocialActivityService.java

```diff
@@ -71,6 +71,17 @@ public long publish(String activity) throws SocialActivityException {
 	}
 
 	/**
+	 * Update an activity given new activity
```

Comment:
Thanks @thushara35 , I have explained it further with [this commit](https://github.com/wso2/carbon-store/pull/844/commits/a6b63663baa40847faa3eb7147aef5c0648d2c59)